tests/bindgen/test_bindgen.py

```diff
@@ -0,0 +1,49 @@
+from . import BindgenTestCase, generate_bindings
+
+def test_bare_funcs():
+    testcase = BindgenTestCase(
+        guest_code_dir='bare_funcs',
+        world_name='barefuncs',
+    )
+    store, root = generate_bindings(testcase)
+    assert root.foo(store, 10) == 11
```

Comment:
Would it be possible to put these individual tests within each folder? For example keeping everything self-contained within each sub-folder to avoid a large listing of all tests here.

Reply:
I'll see if I can get that working. I had to configure pytest to ignore these directories to avoid issues with module name conflicts with `app.py`, but there might be a way around this.  Worst case, we can split them out into separate files in this directory, similar to the existing structure in `codegen/`.